code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -234,6 +235,7 @@
 	slot_flags = ITEM_SLOT_MASK
 	adjusted_flags = ITEM_SLOT_HEAD
 	icon_state = "bandbotany"
+	species_exception = list(/datum/species/golem)
```

Comment:
buff to gang golems smh my head my head

Reply:
The Rolling Stones family is the strongest, toughest gang on the station.